ci-operator/config/coreos/coreos-assembler/coreos-coreos-assembler-main.yaml

```diff
@@ -14,6 +14,10 @@ tests:
   commands: coreos-assembler shell true
   container:
     from: bin
+- as: rhcos
+  commands: curl -sSL -o /tmp/test.sh https://raw.githubusercontent.com/openshift/os/master/ci/build-test-qemu.sh && chmod a+x /tmp/test.sh && /tmp/test.sh https://github.com/openshift/os
```

Comment:
So obviously, this is super ugly.  It's all rather circular because we need the code from openshift/os to test openshift/os...

One thought is that we try harder to move the test logic into coreos-assembler.  Something like "detect Prow and set COSA_SKIP_OVERLAY" automatically, etc?  I guess this is kind of Gangplank territory too.

Reply:
I'm confused by this. I would expect instead a script in cosa which uses the ref under test to map that to the right openshift/os release (e.g. `rhcos-4.8` -> `release-4.8`), and then checking out that openshift/os branch and just runs its `ci/build-test-qemu.sh`. Then we could have additional cosa tests there if we wanted to.

Reply:
To be clear, cool as is too with this to start with! CI coverage is infinitely better than no CI coverage. :)

Reply:
Good point, it would be clearly useful to have branched CI fixed by this too.  Will work on that.